openpype/tools/studio_tools/app.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import os
+from Qt import QtCore
+from Qt import QtWidgets
+from openpype import style
+from openpype.settings import get_project_settings
+from openpype.tools.utils import lib as tools_lib
+
+import scriptsmenu
+import logging
+
+log = logging.getLogger(__name__)
+module = sys.modules[__name__]
+module.window = None
+
+
+class StudioToolsDialog(QtWidgets.QWidget):
+    def __init__(self, parent=None):
+        super(StudioToolsDialog, self).__init__(parent)
+
+        self.resize(400, 300)
+        self.setStyleSheet(style.load_stylesheet())
+
+        project_settings = get_project_settings(os.getenv("AVALON_PROJECT"))
+        host = os.getenv("AVALON_APP")
+        config = project_settings[host]["scriptsmenu"]["definition"]
+        _menu = project_settings[host]["scriptsmenu"]["name"]
+        self.setWindowTitle(_menu)
+
+        title = _menu.title()
+        objectName = _menu.title().lower().replace(" ", "_")
+
+        layout = QtWidgets.QVBoxLayout()
```

Comment:
Just set parent of layout. You don't have to call `self.setLayout(layout)` in that case.
```suggestion
        layout = QtWidgets.QVBoxLayout(self)
```